setup.py

```diff
@@ -19,11 +19,13 @@
 import os
 from setuptools import setup, find_packages
 
+
 def read_version():
-    fn = os.path.join(os.path.dirname(__file__), 'sydent', '__init__.py')
+    fn = os.path.join(os.path.dirname(__file__), "sydent", "__init__.py")
     with open(fn) as fp:
         f = fp.read()
-    return re.search(r"^__version__ = '(.*)'", f).group(1)
+    return re.search(r"^__version__ = \"(.*)\"", f).group(1)
```

Comment:
For clarity, it may be easier to just invert the quotes rather than escaping -- Black will recognize when a single-quoted string contains double-quotes and won't mess with it :)

```suggestion
    return re.search(r'^__version__ = "(.*)"', f).group(1)
```

Reply:
Oh cool-I'm iffy at best on regex so this is helpful to know.